install/build.py

```diff
@@ -147,6 +147,7 @@ def get_compiler_setting(use_hip):
             library_dirs.append(os.path.join(cuda_path, 'lib'))
 
     if rocm_path:
+        include_dirs.append(os.path.join(rocm_path, 'hipfft', 'include'))
```

Comment:
This change is not needed because `hipfft.h` can be found in `/opt/rocm/include`, so the line below is enough to make it locatable.

Reply:
See https://github.com/cupy/cupy/pull/5067#discussion_r610272175 above

Reply:
Please see my reply https://github.com/cupy/cupy/pull/5067/files#r610286064.